app/controllers/DefaultsController.scala

```diff
@@ -34,7 +34,7 @@ case class Defaults(
                      capiLiveUrl: String = "",
                      capiPreviewUrl: String = "",
                      availableTerritories: Iterable[TargetedTerritory] = Nil,
-                     availableEditions: List[EditionDefinition]
+                     availableEditions: List[CuratedPlatformDefinition]
```

Comment:
Should this be `availablePlatforms`?

Reply:
Actually it should be `AvailableTemplates`, I think. But that impacts on frontend as well as backend....